### DISCUSSION TITLE
getServerData cache strategy

### DISCUSSION THREAD

**Discussion by krazyjakee** [gatsbyjs/gatsby#34626] [Help]
How would I go about caching the results of the getServerData function?
Right now if I run my mouse over every link on the page, gatsby will fire off hundreds of prefetch requests. I don't link this functionality but since I can't disable it, I would like to be able to ensure that they are at least cache hits.

Thanks.

**Comment by Charlygnusa** [comment]
Yes I also want to know, maybe we can use redux, like if the data is already in redux, then there is no need to fetch the data again...
But how do I access redux inside the getServerData function, because now all I know is that there is only the context parameter given
